platform/android/java/scripts/publish-module.gradle

```diff
@@ -19,7 +19,55 @@ afterEvaluate {
                 // Mostly self-explanatory metadata
                 pom {
                     name = PUBLISH_ARTIFACT_ID
-                    description = 'Godot Engine Android Library'
+                    description = 'Godot Engine Android Library - Template Build'
+                    url = 'https://godotengine.org/'
+                    licenses {
+                        license {
+                            name = 'MIT License'
+                            url = 'https://github.com/godotengine/godot/blob/master/LICENSE.txt'
+                        }
+                    }
+                    developers {
+                        developer {
+                            id = 'm4gr3d'
+                            name = 'Fredia Huya-Kouadio'
+                            email = 'fhuyakou@gmail.com'
+                        }
+                        developer {
+                            id = 'reduz'
+                            name = 'Juan Linietsky'
+                            email = 'reduzio@gmail.com'
+                        }
+                        developer {
+                            id = 'akien-mga'
+                            name = 'Rémi Verschelde'
+                            email = 'rverschelde@gmail.com'
+                        }
+                        // Add all other devs here...
```

Comment:
Listing all Godot developers here seems a bit unrealistic.

Couldn't we just have `Godot Engine contributors` with an email like `contact@godotengine.org`?

Reply:
@akien-mga I've addressed the feedback in https://github.com/godotengine/godot/pull/105611